src/composables/trade/useSor.ts

```diff
@@ -32,6 +32,12 @@ const GAS_PRICE = process.env.VUE_APP_GAS_PRICE || '100000000000';
 const MAX_POOLS = process.env.VUE_APP_MAX_POOLS || '4';
 const SWAP_COST = process.env.VUE_APP_SWAP_COST || '100000';
 const MIN_PRICE_IMPACT = 0.0001;
+const HIGH_PRICE_IMPACT_THRESHOLD = 0.05;
+const INITIAL_STATE = {
+  errors: {
+    highPriceImpact: false
+  }
+};
```

Comment:
Again you could be using the reactive state pattern here, but you could also actually include all the composable state in one reactive object.

Reply:
Yeah good call!